tailwind.config.js

```diff
@@ -1,5 +1,5 @@
 const BASE_FONT_SIZE = 16;
-const pxToRem = (px) => `${ px / BASE_FONT_SIZE }rem`;
+const pxToRem = (px) => `${px / BASE_FONT_SIZE}rem`;
```

Comment:
I guess the built in prettier makes this happen, try to turn this feature off, because it removes the spaces from within the curly braces. also put back spaces here :)